src/libs/ReportUtils.js

```diff
@@ -543,6 +543,20 @@ function hashLogin(login, range) {
     return Math.abs(hashCode(login.toLowerCase())) % range;
 }
 
+/**
+ * There is a possibility that if the Expense report has a negative total.
+ * This is because there are instances where you can get a credit back on your card,
+ * or you enter a negative expense to “offset” future expenses
+ *
+ * @param {Object} moneyRequestReport
+ * @returns {Number}
+ */
+function getMoneyRequestAmount(moneyRequestReport) {
+    const moneyRequestAmount = lodashGet(moneyRequestReport, 'total', 0);
```

Comment:
If the amount is 0, we should return early. Otherwise we might end up with -0
 
<img width="110" alt="Screenshot 2023-05-12 at 12 45 45 AM" src="https://github.com/Expensify/App/assets/22219519/0dfbbf4c-2a0e-46ce-be0c-7c17ece1c6c1">